versions/3.1.1.md

```diff
@@ -2346,6 +2346,15 @@ While composition offers model extensibility, it does not imply a hierarchy betw
 To support polymorphism, the OpenAPI Specification adds the `discriminator` keyword.
 When used, the `discriminator` indicates the name of the property that hints which schema definition is expected to validate the structure of the model.
 
+###### Generic (Template) Data Structures
+
+Implementations MAY support defining generic or template data structures using JSON Schema's dynamic referencing feature:
```

Comment:
Does Swagger UI support these already?

Reply:
No, why should that matter?  Sswagger is the slowest-moving major tooling vendor out there.

Reply:
Actually I'm just assuming "no" - the whole point of this change is to raise awareness that this works – please see the _six different issues requesting this feature_ linked in the issue for this PR.  I'll list them here for convenience- I closed them becauase it's technically possible, but got feedback that it's not obvious enough:

* #519 
* #555 
* #773 
* #957 
* #1749 
* #2453

Reply:
My experience is that what doesn't show up in Swagger UI isn't written into an OpenAPI description. Which shouldn't prevent us from advertising new features 😄

Reply:
@ralfhandl I'm glad you're not seeing it as a barrier :-)  But in all seriousness, there are other tooling vendors, and they do have actual customers/users.  I really don't think it's a good idea for the steering committee of a vendor-neutral project to publicly promote the idea that one company's single product is the gating factor for the entire ecosystem.  We've been working very hard for years now to separate "OpenAPI" from "Swagger", and comments like this do the opposite.